src/main/java/net/sf/jabref/logic/specialfields/SpecialFieldsUtils.java

```diff
@@ -0,0 +1,139 @@
+package net.sf.jabref.logic.specialfields;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import net.sf.jabref.logic.util.UpdateField;
+import net.sf.jabref.model.FieldChange;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.model.entry.Keyword;
+import net.sf.jabref.model.entry.KeywordList;
+import net.sf.jabref.model.entry.specialfields.SpecialField;
+import net.sf.jabref.model.entry.specialfields.SpecialFields;
+
+@Deprecated // the class should be refactored and partly integrated into BibEntry
```

Comment:
Please add a deprecated javadoc comment, too.
http://stackoverflow.com/questions/5039723/should-i-use-javadoc-deprecation-or-the-annotation-in-java